docs/src/gettingstarted/introduction/template/cli-install.md

```diff
@@ -14,63 +14,32 @@ With all of the requirements met, install the CLI to start developing with Platf
 
 1. **Install the CLI**
 
-    In your terminal, run the following command depending on your operating system:
-
-{{< codetabs >}}
-
----
-title=Linux/macOS
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -fsS https://platform.sh/cli/installer | php
-
-<--->
-
----
-title=Windows
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -f https://platform.sh/cli/installer -o cli-installer.php
-php cli-installer.php
-
-{{< /codetabs >}}
+  {{% cli-installation %}}
 
 
 2. **Authenticate and Verify**
 
-   Once the installation has completed, you can run the CLI in your terminal with the command
-
-   ```bash
-   platform
-   ```
-
-   {{< note >}}
-
-   If you opened your free trial account using another login (such as GitHub),
-   you can't authenticate with this command until you setup your account password with Platform.sh in the console.
-
-   {{< /note >}}
+  {{% cli-installation-check %}}
 
-   You should now see a list of your Platform.sh projects, including the template you made in this guide.
-   You can copy its *project ID* hash, and then download a local copy of the repository with this command:
+    {{< note >}}
+    
+    If you opened your free trial account using another login (such as GitHub),
+    you can't authenticate with this command until you setup your account password with Platform.sh in the console.
+    
+    {{< /note >}}
```

Comment:
:question: 
Is this true? I use Google and I'm not sure I've ever set up an account password in the console.

If it is true, shouldn't it be other places as well? Seems like this isn't the only place it'd apply.

Reply:
Good catch, I created a new trial user, and indeed, that part is superfluous.